app/models/tenant.rb

```diff
@@ -290,6 +291,34 @@ def allowed?
     Rbac::Filterer.filtered_object(self).present?
   end
 
+  def create_miq_product_features_for_tenant_nodes
+    result = MiqProductFeature.with_tenant_node_features.map.each do |tenant_miq_product_feature|
+      create_miq_product_feature(tenant_miq_product_feature)
+    end.flatten
+
+    result = MiqProductFeature.create(result).map(&:identifier)
+
+    MiqProductFeature.invalidate_caches
+    result
+  end
+
+  def create_miq_product_feature(miq_product_feature)
```

Comment:
This should be named `build_...` instead of `create_...` since it doesn't actually create the `miq_product_feature`